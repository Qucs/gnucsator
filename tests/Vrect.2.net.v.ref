--- Vrect.2.net.v.out	2018-09-19 02:01:31.275945858 +0100
+++ Vrect.2.net.v.ref	2018-09-19 01:49:18.763975897 +0100
@@ -1,3 +1,4 @@
 
+vsource #(.dc(0.)) p (.p(gnd),.n(0));
 Vprobe #() Pr1 (.p(_net1),.n(0));
 Vrect #(.U(1),.Td(0n),.Tr(1n),.Tf(1n),.TH(1m),.TL(1m)) V1 (.p(_net1),.n(0));
