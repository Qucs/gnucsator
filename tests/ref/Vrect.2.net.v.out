
module main ();
//# Qucs 0.0.19  /home/felix/extern/gnucap-plugins/tests/simple.sch 
VProbe #() Pr1 (.p(_net1),.n(0));
Vrect #(.U(1),.Td(0n),.Tr(1n),.Tf(1n),.TH(1m),.TL(1m)) V1 (.p(_net1),.n(0));
endmodule // main

